tests/compare/README.md

```diff
@@ -0,0 +1,23 @@
+# Brotli Compare
+Basic scripts to test any file comparing Gzip and Brotli across speed and compression strength.  Output is dumped to an HTML result and plotted with Google Charts.  It can be easily extended or changed to compare other compression engines.  Note that default Brotli compression level (MAX) is not tested as it skews the graph for time but it's worth considering manually.
+
+## Usage
+```
+$ git clone 
+$ ./compare.sh [test file]
```

Comment:
I believe, the path is a little bit longer =)

Reply:
Yeah sorry I developed it at root of a repo I created. Did you try it out anyway? I can correct the readme or play the pseudocode card.